.irb_config

```diff
@@ -0,0 +1,11 @@
+# Not that we use a non-standard config file name to reduce
+# name clashes with other IRB config files like `.irbrc`.
+
+require 'irb/completion'
+
+irb_history = (HOMEBREW_REPOSITORY/".irb_history")
```

Comment:
Feels a bit weird having this in the repository rather than `$HOME`? Is this because the sandbox will let you write in the prior and not the latter?

Reply:
If we want the history to be exclusive to Homebrew, I think it makes sense to put it in that directory. I don't think it's a good idea or expected behavior to interact with the global IRB history though if that's what you were suggesting.

Reply:
@apainintheneck Feels weird to put something user-specific like this in the repository. If there's no technical reason against it being e.g. `$HOME/.brew_irb_history`: I think that'd be preferred.

Reply:
It works that way too. Updated in https://github.com/Homebrew/brew/pull/14892/commits/deabd4a5f9adb3fa51ac732c86610c700b52e96d